### PR TITLE
feat: replaced "uuid" with "crypto.randomUUID()" (#470)

### PR DESCRIPTION
**Pull request by biplobsd**
I have deleted all uuid and used crypto.randomUUID(). I also deleted uuid file for is being unused function.

Fixed : #470

### CODE REVIEW

File: src/interceptors/ClientRequest/NodeClientRequest.ts [hunk 1]
```diff
@@ -150,7 +149,7 @@ export class NodeClientRequest extends ClientRequest {
   end(...args: any): this {
     this.logger.info('end', args)
 
-    const requestId = uuidv4()
+    const requestId = crypto.randomUUID()
```
Comment (kettanaito) [inline src/interceptors/ClientRequest/NodeClientRequest.ts:152]: Can you please import `crypto` from `node:crypto`? Afaik, it's not a universally supported global yet.

Comment (biplobsd) [reply src/interceptors/ClientRequest/NodeClientRequest.ts:152]: Ok.

**Comment by biplobsd** [comment]
Hello @kettanaito, is there any changes needed? 😐

**Comment by hpohlmeyer** [comment]
Just saw this PR and wanted to note, that this makes @mswjs/interceptors incompatible with browsers. If I remember correctly, msw uses the interceptors library to defer request execution until the service worker has been registered.

`node:crypto` is not available in browsers, so this would break compatibility. But even using `crypto.randomUUID()` without the import breaks compatibility, if it is used on a http site, because `crypto` is only available on https sites.

**Comment by biplobsd** [comment]
> Just saw this PR and wanted to note, that this makes @mswjs/interceptors incompatible with browsers. If I remember correctly, msw uses the interceptors library to defer request execution until the service worker has been registered.
> 
> `node:crypto` is not available in browsers, so this would break compatibility. But even using `crypto.randomUUID()` without the import breaks compatibility, if it is used on a http site, because `crypto` is only available on https sites.

Okay, I'm close this PR. @hpohlmeyer . This changes make more problem 😄

**Pull request closed without merging**